shared/src/commonMain/kotlin/co/touchlab/kampkit/models/BreedRepository.kt

```diff
@@ -11,15 +10,16 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.flow.mapNotNull
 import kotlinx.datetime.Clock
-import org.koin.core.component.KoinComponent
-import org.koin.core.component.inject
 
-class BreedModel : KoinComponent {
-    private val dbHelper: DatabaseHelper by inject()
-    private val settings: Settings by inject()
-    private val dogApi: DogApi by inject()
-    private val log: Logger by injectLogger("BreedModel")
-    private val clock: Clock by inject()
+class BreedRepository(
+    private val dbHelper: DatabaseHelper,
+    private val settings: Settings,
+    private val dogApi: DogApi,
+    log: Logger,
+    private val clock: Clock
+) {
+
+    private val log = log.withTag("BreedModel")
```

Comment:
Don't need this anymore as the tag is added [here](https://github.com/touchlab/KaMPKit/pull/231/files#diff-cbdbeddd2829ea6c2955853cd60a5b36fbca7226a2473d3287a101d19a879d3bR73), right?